Shared/Samples/Change map view background/README.md

```diff
@@ -0,0 +1,34 @@
+# Change map view background
+
+Customize map view's background by changing its grid properties.
+
+![Image of change map view background](change-map-view-background-settings.png)
+
+## Use case
+
+A background grid defines the default color and grid for display behind a map or scene surface. Set properties on the background grid to highlight and give context to your MapView or SceneView.
```

Comment:
```suggestion
A background grid defines the default color and grid for display behind a map or scene surface. Set properties on the background grid to highlight and give context to your map view or scene view.
```
Typically unless necessary, we don't use the symbol names until `How it works` section.